packages/permission-controller/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.0]
+### Changed
+- Restrict certain permissions by subject type ([#1233](https://github.com/MetaMask/core/pull/1233))
+- Move `SubjectMetadataController` to permission-controller package ([#1234](https://github.com/MetaMask/core/pull/1234))
+- deps: eth-rpc-errors@4.0.0->4.0.2 ([#1215](https://github.com/MetaMask/core/pull/1215))
```

Comment:
We usually don't include dep changes unless there is a big change for consumers. This isn't one I think?

Reply:
Agreed that `package.json` changes are nice to include. Its the lockfile-only changes that are unnecessary to add, because they don't impact the users of the package at all.

I would have assumed this one was a lockfile change because 4.0.2 is in-range of 4.0.0, but I see that it bumped the minimum version in `package.json` as well.

Reply:
Nit: Perhaps rewording it to highlight how this affects users of the package would make this more clear, e.g.

```suggestion
- Update minimum `eth-rpc-errors` version from `4.0.0` to `4.0.2` ([#1215](https://github.com/MetaMask/core/pull/1215))
```

Reply:
@legobeat since you've accepted this suggestion, would you mind copying it to the other changelog entries as well? This change entry is present in a few other packages as well